index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300&display=swap" rel="stylesheet">
+
+    <title>Evolo</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo-container">
+
+            <p>Evolo</p>
+
+        </div>
+        <nav>
+            <ul>
+                <li>Home</li>
```

Comment:
Each list item must be a link. So, add an anchor for each one. 

`            <li><a href="#">Home</a></li>
`